solarwinds_apm/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.7.0"
+__version__ = "2.0.0.0"
```

Comment:
Is it a typo?

Reply:
Nope, I do test releases with a 4th "dev" digit